lms/static/scripts/frontend_apps/utils/initial-loading-context.ts

```diff
@@ -0,0 +1,28 @@
+import { createContext } from 'preact';
+
+import type { ErrorLike } from '../errors';
+
+export type InitialLoadingContextState = {
+  /**
+   * Report the beginning of a loading that needs to be taken into
+   * consideration for the initial load.
+   *
+   * @param id - The unique identifier for this loading instance
+   */
+  startLoading: (id: string) => void;
```

Comment:
This API uses the parameter name `id` but the functions in `HideUntilLoad` use the argument name `key`.